x/distribution/keeper/keeper.go

```diff
@@ -38,7 +39,7 @@ func NewKeeper(
 	cdc codec.BinaryCodec, storeService store.KVStoreService,
 	ak types.AccountKeeper, bk types.BankKeeper, sk types.StakingKeeper,
 	feeCollectorName, authority string,
-) Keeper {
+) *Keeper {
```

Comment:
why is this changed to a pointer? is it bc of carbon `app.go` related dependency init?

Reply:
Yeah and JR wanted the assigning of distr hooks to be done with pointers